charts/sn-platform/templates/streamnative-console/streamnative-console-initialize.yaml

```diff
@@ -33,6 +33,9 @@ spec:
       labels:
         {{- include "pulsar.template.labels" . | nindent 8 }}
     spec:
+      {{- if .Values.global.imagePullSecrets }}
+      imagePullSecrets: {{- toYaml .Values.global.imagePullSecrets | nindent 8 }}
```

Comment:
It would be better to simplify it by removing the `if` condition check, just use `{{- toYaml .Values.global.imagePullSecrets | nindent 8 }}`.

Reply:
This would cause wrong rendering as shown below without the `if` block. 
The `imagePullSecrets` is set with a default value so that we have to check it.

```
spec:
  imagePullSecrets:
    []
```